uzdevums_2.py

```diff
@@ -0,0 +1,6 @@
+# Definēt divus mainīgos a = 5, b = 9 un izvadīt atlikumu b dalot ar a.
+
+a = 5
+b = 9
+product = b / a
```

Comment:
Atlikumu iegūst ar `%` darbību:
```suggestion
product = b % a
```